src/styles/mixins/_overlay.scss

```diff
@@ -46,5 +42,5 @@
  * - FTUX tooltip guides
  */
 @mixin bdl-Overlay-containerShadow {
-    box-shadow: 0 $bdl-grid-unit ($bdl-grid-unit * 3) 0 rgba($black, .1);
+    box-shadow: 0 $bdl-grid-unit ($bdl-grid-unit * 3) 0 rgba($black, .12);
```

Comment:
@naeims this is the one i was talking about. i would expect box shadow colors not to vary by so little